treeherder/config/settings.py

```diff
@@ -27,11 +27,11 @@
 GRAPHQL = env.bool("GRAPHQL", default=True)
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY")
+SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY", default='secret-key-of-at-least-50-characters-to-pass-check-deploy')
 
 # Hosts
 try:
-    SITE_URL = env("SITE_URL")
+    SITE_URL = env("SITE_URL", default='http://localhost:8000')
```

Comment:
This is a comment for my own self: Keep an eye on this differing value `SITE_URL=http://backend:8000/`